index.html

```diff
@@ -4479,6 +4479,50 @@ <h2>HTTP</h2>
         <section class="informative">
           <h2>Credential Type-Specific Processing</h2>
 
+          <p>
+RDF is a technology for modeling graphs of statements. Each statement is a
+single <i>subject&#x2192;property&#x2192;value</i> (also known as
+<i>entity&#x2192;attribute&#x2192;value</i>) relationship, which is referred to
+as a <a>claim</a> in this specification. JSON-LD is a technology that enables
+the expression of RDF using idiomatic JSON, enabling developers familiar with
+JSON to write applications to consume RDF as JSON. In general, subjects are
+expressed as JSON objects with each property and value of the subject as a JSON
+key and value, respectively. A special affordance is made to express an
+identifier of a subject, if present, using the <code>@id</code> JSON key, which
+is aliased in this specification to <code>id</code>. See
+<a data-cite="?JSON-LD11#relationship-to-rdf">Relationship to RDF</a> for more
+details.
+          </p>
+
+          <p>
+As JSON can be used to express all different kinds of information, a consumer of
+a particular JSON document can only properly interpret the authors intent if they
+possess information that contextualizes and disambiguates it from other possible
+expressions. Information to assist with this interpretation can either be wholly
+external to the JSON document or linked from within it. Compact JSON-LD documents
+include a <code>@context</code> property that internally expresses or links to
+contextual information and, per the JSON-LD specification, these documents follow
+object-oriented organizational principles to express <a>claims</a>. These features
```

Comment:
I am not sure what "object-oriented organizational principles" means. More exactly... I of course know that we refer to issues like using `@protected`, local `@context`, etc., but these may be meaningless for a casual reader, who may associate these terms to programming language classes, their class inheritance, properties, etc. 

I think this side remark can be safely removed.

```suggestion
contextual information to express the correct <a>claims</a>. These features
```

Reply:
I'm fine to take it out, but I think the original intent was to highlight in this context what RDF is providing "under-the-hood". @dlongley iirc, noting this was your suggestion, so your input here would be helpful before I take it out. 😃

Reply:
Yeah, the question is who the target is for this. My assumption is that the target is those who do not really know and, actually, do not really care about the RDF aspect, just want to get things work for them without too many questions asked. For an audience who know RDF & Co, the whole text is, I believe, unnecessary anyway...

Reply:
I think this is good text to help people understand that statements aren't expressed in either a flat or bespoke format, but JSON objects really are used to express objects (things) where each JSON key is a property and value of said thing. You can design your JSON in totally different ways from this where no information can be gleaned from the JSON structure alone -- but JSON-LD requires it to work in a way that is true to this object oriented nature.

So I'd prefer to keep it; I don't think we come out and say this anywhere else and it's helpful to understand this overarching principle in how JSON-LD expresses claims.

Reply:
Though not convinced, I won't lie down the road on this.

Can we at least find another formulation? "Object-oriented organizational principles" bothers my ears, to be honest...

Reply:
@iherman I've come around to your way of thinking. I think the current statement is a bit non sequitur as well.

@dlongley if it's important to express the OOP nature of JSON-LD vs. random JSON structures, I think we need to do it elsewhere in this text. Happy to take a crack at it, if you think it's important.

Reply:
If we want to strike that one phrase and add some other text to address it in another way in a follow-on PR, that's fine. I don't want to hold up the other good text here.

Reply:
FWIW, I agree with Ivan... it's not clear what "object-oriented organizational principles" means... is it talking about encapsulation? abstractions? design patterns? I suggest just striking ", per the JSON-LD specification, these documents follow object-oriented organizational principles to express <a>claims</a>" (because we already explain that several times throughout the specification, starting in the "Core Data Model" section.

Reply:
It has been removed from the latest iteration, as far as I can see, so marking this as resolved.